lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -470,7 +524,8 @@ SingleExitLoopTransformer::createNewHeader() {
   }
   header->dropAllArguments();
   // Add phi arguments in the new header corresponding to the escaping values.
-  for (const auto &escapingValue : escapingValues) {
+  for (const auto &kv : escapingValueSubstMap) {
+    const SILValue& escapingValue = kv.first;
```

Comment:
i believe SILValue is a lightweight wrapper, and we need not take a reference. (making a copy is fine)

Reply:
Done.